examples/01_Simulation.ipynb

```diff
@@ -1280,39 +1342,533 @@
+    "    plt.plot(simulated_results.times, third_throw_v, color='tab:cyan')\n",
+    "    \n",
+    "    plt.plot(simulated_results.times, fourth_throw_x, color='tab:purple', linestyle='dashed')\n",
+    "    plt.plot(simulated_results.times, fourth_throw_v, color='tab:purple')\n",
+    "    \n",
+    "    plt.xlabel(\"time (s)\")\n",
+    "    plt.ylabel(\"state\") \n",
+    "\n",
+    "    color_legend = [Line2D([0], [0], marker='o', markersize=10, color='tab:blue', linestyle='None',label='Throw 1'),\n",
+    "                    Line2D([0], [0], marker='o', markersize=10, color='tab:orange', linestyle='None', label='Throw 2'),\n",
+    "                    Line2D([0], [0], marker='o', markersize=10, color='tab:cyan', linestyle='None', label='Throw 3'),\n",
+    "                    Line2D([0], [0], marker='o', markersize=10, color='tab:purple', linestyle='None', label='Throw 4')]\n",
+    "\n",
+    "    linestyle_legend = [Line2D([0], [0], color='black', lw=2, linestyle='-', label='Position (x) [m]'),\n",
+    "                    Line2D([0], [0], color='black', lw=2, linestyle='--', label='Velocity (v) [m/s]')]\n",
+    "\n",
+    "    plt.legend(handles=color_legend + linestyle_legend, bbox_to_anchor=(1,1))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Noise\n",
-    "**A version of this section will be added in release v1.8**"
+    "We will now set up the vectorized initial state. In this example, we will define 4 throws of varying positions (x) and strengths (v) in `first_state`. We will then simulate to the threshold using this state. We should see the simulation stop once all objects hits the ground since the `event_strategy` is 'all'."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "first_state = {\n",
+    "    'x': array([1.75, 1.8, 1.85, 1.9]),\n",
+    "    'v': array([35, 39, 22, 47])\n",
+    "}\n",
+    "\n",
+    "m = ThrownObject()\n",
+    "simulated_results = m.simulate_to_threshold(x=first_state, events='impact', event_strategy='all', print=True, dt=0.1, save_freq=1)\n",
+    "\n",
+    "print_vectorized_sim_plots(simulated_results)\n",
+    "plt.title(\"Vectorized simulation until any object hits the ground\", pad=10)\n",
+    "plt.show()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Vectorized Simulation\n",
-    "**A version of this section will be added in release v1.8**"
+    "## Configuring Simulation"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Configuring Simulation\n",
-    "**A version of this notebook will be added in release v1.8, including:**\n",
-    "* t0, x\n",
-    "* integration_method"
+    "We can explore other ways to configure simulations. We will use the Battery model for these examples."
```

Comment:
```suggestion
    "Here we will explore other ways to configure simulations. These approaches can be used to configure a simulation even further to match a use case. We will use the Battery model for these examples."
```

Might be better wording than this but I was trying to add some more intro for the section